Session/Settings/DeveloperSettings/DeveloperSettingsViewModel.swift

```diff
@@ -42,6 +42,7 @@ class DeveloperSettingsViewModel: SessionTableViewModel, NavigatableStateHolder,
         case logging
         case network
         case disappearingMessages
+        case conversationBanners
```

Comment:
Can probably just add this one to the `general` section rather than create a new section